packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -138,6 +141,11 @@ const SUSPENSE_END_DATA = '/$';
 const SUSPENSE_PENDING_START_DATA = '$?';
 const SUSPENSE_FALLBACK_START_DATA = '$!';
 
+const INSERTION_MARKER = '%';
+
+const HTML = 'html';
+const HEAD = 'head';
+const BODY = 'body';
 const STYLE = 'style';
```

Comment:
This was originally only added so that if we do things like `props[STYLE]` it would be known to be treated as a dictionary look up (by Google Closure Compiler and just in terms of code so you think about optimizing it for dictionary look ups instead of hidden class directly field access).

When you're comparing to a hard coded string, you don't need this. Just inline the string.

Reply:
I learned this looking at the diff of bundle size. i was surprised to see the build undid the indirection and made everything strings but your explanation tells me why :)